### ISSUE TITLE
Save data normalization parameters

### ISSUE THREAD

**Issue by sjawhar** [sjawhar/no-wander#11]
Currently, using `--preprocess normalize` results in a simple call to `sklearn.preprocessing.robust_scale`:
https://github.com/sjawhar/no-wander/blob/c74c5d525c50a93a061d86727ddf78b5a007b383/app/src/features.py#L200-L203

However, normalization parameters need to be saved for reuse with test set (e.g. by using `RobustScaler` instead).
